nixos/tests/podman.nix

```
@@ -0,0 +1,30 @@
+# This test runs podman and checks if simple container starts
+
+import ./make-test-python.nix (
+  { pkgs, ... }: {
+    name = "podman";
+    meta = with pkgs.stdenv.lib.maintainers; {
+      maintainers = [ adisbladis ];
```

Could use the team here as well?

```suggestion
    meta = with pkgs.stdenv.lib {
      maintainers = teams.podman.members;;
```